ui/partials/perf/comparechooserctrl.html

```diff
@@ -16,7 +20,11 @@
           <label for="new-project-selector">Project</label>
           <select id="new-project-selector" class="form-control" ng-model="newProject" ng-options="project.name for project in projects"/>
           <label for="new-revision-input">Revision</label>
-          <input id="new-revision-input" maxlength="12" class="form-control" ng-model="newRevision">
+            <div class="form-group has-error">
+            <input id="new-revision-input" maxlength="12" class="form-control" ng-model="newRevision">
+            <br/>
```

Comment:
If these two `input` and `br` lines are child elements of the parent div, I believe they should be indented 2 chars.